webapp/src/main/resources/sass/mojito.scss

```diff
@@ -1178,4 +1169,18 @@ label.current-pageNumber {
         background-position: -300% center;
     }
 
+}
+
+.break-word {
+    word-break: break-word;
+}
+
+.break-word-ltr-safe {
```

Comment:
What is `ltr` referring to here, I'm assuming left to right? If so I don't see any LTR language specific handling for when it's being applied?

Reply:
Yes, ltr in this context does refer to left-to-right here. 

There isn't any explicit ltr handling, it uses different CSS properties to tell the container how to handle the text overflowing. The previous style breaks the characters in the unicode string but for languages using more complex encodings, breaking the character over different lines might not be possible. So this tells the container to arrange itself such that the resulting text is wrapped along the layout / container and not done.

But I realize now from your comment that the name is a bit ambiguous as this actually handles both ltr and rtl cases.